models/project/project.go

```diff
@@ -6,6 +6,7 @@ import (
 	"../../modules/logging"
 	"../db"
 	"../list"
+	"../list_option"
```

Comment:
__[gometalinter]__
* [gotype] could not import ../list_option (can't find import: /sideci/fascia/models/list_option)